internal/database/models/exploit_model.go

```diff
@@ -17,19 +17,64 @@ package models
 import "time"
 
 type Exploit struct {
-	ID          int        `json:"id" gorm:"primaryKey;"`
-	Published   *time.Time `json:"published" gorm:"type:date;"`
-	Updated     *time.Time `json:"updated" gorm:"type:date;"`
+	ID          string     ` gorm:"primaryKey;"`
+	LocalID     int        `json:"id" gorm:"-"`
+	Published   *time.Time `json:"pushed_at" gorm:"type:date;"`
+	Updated     *time.Time `json:"updated_at" gorm:"type:date;"`
 	Author      string     `json:"author" gorm:"type:text;"`
 	Type        string     `json:"type" gorm:"type:text;"`
 	Verified    bool       `json:"verified" gorm:"type:boolean;"`
-	SourceURL   string     `json:"sourceURL" gorm:"type:text;"`
+	SourceURL   string     `json:"html_url" gorm:"type:text;"`
 	Description string     `json:"description" gorm:"type:text;"`
 	CVE         CVE        `json:"cve" gorm:"foreignKey:CVEID;constraint:OnDelete:CASCADE;"`
 	CVEID       string     `json:"cveID" gorm:"type:text;"`
 	Tags        string     `json:"tags" gorm:"type:text;"`
+	Owner       Owner      `json:"owner" gorm:"-"`
+	Forks       int        `json:"forks" gorm:"type:integer;"`
+	Watchers    int        `json:"watchers" gorm:"type:integer;"`
+	Subscribers int        `json:"subscribers_count" gorm:"type:integer;"`
+	Stars       int        `json:"stargazers_count" gorm:"type:integer;"`
+}
+
+type Owner struct {
+	Login string `json:"login"`
 }
 
 func (m Exploit) TableName() string {
 	return "exploits"
 }
+
+/*
+// Exploit represents a GitHub exploit
```

Comment:
this is a great comment! Please keep that. 

```suggestion
// GitHub exploit json
```